conn.go

```diff
@@ -60,6 +62,14 @@ func (c *Conn) RemoteAddr() net.Addr {
 	}
 }
 
+func (c *Conn) tagHop() {
+	c.relay.host.ConnManager().UpsertTag(c.stream.Conn().RemotePeer(), "relay-hop-stream", incrementTag)
```

Comment:
In a `A --> R --> B` interaction, this `Conn` object represents `A --> R` and `B --> R`. For every connection we make to the relay, we are boosting the hop relay score, to tell the connection manager we are relying on that relay for connectivity to other peers.